glsk/glsk-document-api/src/main/java/com/powsybl/glsk/api/util/converters/GlskPointLinearGlskConverter.java

```diff
@@ -35,9 +36,9 @@ private GlskPointLinearGlskConverter() {
      * @param glskPoint GLSK Point
      * @return farao-core LinearGlsk
      */
-    public static LinearGlsk convert(Network network, AbstractGlskPoint glskPoint) {
+    public static SensitivityVariableSet convert(Network network, AbstractGlskPoint glskPoint) {
 
-        Map<String, Float> linearGlskMap = new HashMap<>();
+        List<WeightedSensitivityVariable> weightedSVs = new ArrayList<>();
```

Comment:
we prefer in general an explicit name for variables. You can use here `weightedSensitivityVariables`.